frontend/src/views/main/study/Decks.vue

```diff
@@ -75,6 +92,7 @@
     selected: IComponents["Deck"][] = [];
     studyNumOptions: number[] = [5, 10, 20, 30, 50];
     selectedNum = 20;
+    vocabIdentifier = process.env.VUE_APP_VOCAB_DECK + " (Can only be studied on its own!)";
```

Comment:
I'm thinking that it might actually be good for us to add a new `deck_type` called `vocab`, this will allow us to more easily filter for studies of the vocab desk and allow us to add additional decks beyond GRE in the future? It means also we don't have to hardcode in this way. 

Also, what was the reason again for not wanting to allow more than one deck studied at the same time?

Reply:
Add a new deck type called public_mnemonic!